### PR TITLE
[MM-34068] Add cws compliance cronjob and secret updates.

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
- Add CWS compliance cronjob and secret updates

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-34068

### CODE REVIEW

File: terraform/aws/modules/customer-web-server/cws_compliance.tf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  count = var.deploy_cws_compliance_cronjob ? 1 : 0
+
+  metadata {
+    name      = "cws-compliance-lookup"
+    namespace = var.cws_name
+  }
+  spec {
+    concurrency_policy            = "Forbid"
+    failed_jobs_history_limit     = 2
+    successful_jobs_history_limit = 2
```
Comment (stafot) [inline terraform/aws/modules/customer-web-server/cws_compliance.tf:11]: @stylianosrigas vvvvvvvv minor But do we actually need more than 1?

**Review by stafot — APPROVED** [review]
LGTM

File: terraform/aws/modules/customer-web-server/cws_compliance.tf (hunk 1 above)
Comment (stylianosrigas) [reply terraform/aws/modules/customer-web-server/cws_compliance.tf]: Thanks makes sense. It runs only every 12 hours so we don't need more that one ;)

**Review by stafot — APPROVED** [review]

**Review by angeloskyratzakos — APPROVED** [review]
LGTM

**Pull request merged**